common/service/config/log.go

```diff
@@ -38,7 +38,7 @@ const fileMode = os.FileMode(0644)
 func (cfg *Logger) NewZapLogger() *zap.Logger {
 	encodeConfig := zapcore.EncoderConfig{
```

Comment:
like I said above, set the default value here:
```
levelKey := s.cfg.Log.LevelKey
if levelKey == ""{
    levelKey = "level"
}
```

Reply:
done